### PR TITLE
Story block: Make stories available in the block inserter

### PR DESCRIPTION
**Pull request by Tug**
#### Changes proposed in this Pull Request:
This is the release PR for the story block. It just flips the switch and makes the block available from the inserter in production.

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:

* Run `yarn build-production`
* Load the editor, make sure the story block is available from the inserter and works as expected

### CODE REVIEW

**Review by aforcier — DISMISSED** [review]
Working great! Tested with a production Jetpack build and on WP.com. :shipit: 🎉

**Review by jeherve — CHANGES_REQUESTED** [review]

File: projects/plugins/jetpack/extensions/blocks/story/index.js [hunk 1]
```diff
@@ -58,7 +57,7 @@ export const settings = {
 	attributes,
 	supports: {
 		html: false,
-		inserter: process.env.NODE_ENV !== 'production' || Platform.OS !== 'web', // false for web, true for mobile
+		inserter: true,
```
[the remark this answers is not in the document]
Comment (Tug) [reply projects/plugins/jetpack/extensions/blocks/story/index.js]: 👍

**Comment by Tug** [comment]
I have addressed your feedback @jeherve. Btw let's hold off the merge for a bit on this one as I'd like https://github.com/Automattic/jetpack/pull/19876 to land first.

**Review by jeherve — APPROVED** [review]
Ready to merge whenever you are ready to push the button.

**Review by aforcier — APPROVED** [review]

**Comment by Tug** [comment]
As discussed on other channels with @aforcier let's not wait for #19876 which will need extensive testing before it gets merged.

**Comment by Tug** [comment]
Deployed to WPCOM with changeset ID 226118

**Pull request merged**